libcaf_net/caf/net/udp_datagram_socket.hpp

```diff
@@ -56,9 +56,23 @@ error allow_connreset(udp_datagram_socket x, bool new_value);
 variant<std::pair<size_t, ip_endpoint>, sec> read(udp_datagram_socket x,
                                                   span<byte> buf);
 
+/// Sends the content of `bufs` as a datagram to the endpoint `ep` on socket
+/// `x`.
+/// @param x x The UDP socket for sending datagrams.
+/// @param bufs Points to the datagram to send, scattered across up to 10
+///             buffers.
+/// @param ep The enpoint to send the datagram to.
+/// @returns The number of written bytes on success, otherwise an error code.
+/// @relates udp_datagram_socket
+/// @post either the result is a `sec` or a positive (non-zero) integer
```

Comment:
```suggestion
```

This adds nothing. The type `variant<size_t, sec>` already says that and `@returns` already documents what the integer is there for.